README.rst

```diff
@@ -28,28 +28,46 @@ Building is as simple as
         $ make       # Let make do the heavy work.
 
 
+CMake accepts the following options:
+
+================= ======= ==========================================================================
+Name              Default Explanation
+================= ======= ==========================================================================
+OVERRIDE_VCS      OFF     Override the lack of a VCS checkout.
+ASAN              OFF     Use AddressSanitizer (https://clang.llvm.org/docs/AddressSanitizer.html).
+USERDATA_PREFIX   '.'     Directory where user data such as savegames is stored.
+================= ======= ==========================================================================
+
 -  **src** directory contains the source code of the FreeRCT program itself.
 -  **src/rcdgen** directory contains the source code of the *rcdgen* program, that builds RCD files from source (which are read by *freerct*).
 - **graphics/rcd** directory contains the source files of the RCD data files, except the graphics.
 - **graphics/sprites** directory contains all the graphics of the game.
 - **bin** directory contains the actual *freerct* executable along with some other files required to actually run the program.
+- **data** directory contains miscellaneous assets loaded by FreeRCT at runtime.
+- **utils** directory contains miscellaneous utility programs and scripts that may make developers' lives easier.
 
 The *cmake/make* commands above will generate the *rcdgen* program, the rcd files and build the 'freerct' program in the src directory.
 
 Config file
 -----------
 
-Finally, you need a 'freerct.cfg' INI format file next to the 'freerct' program in the **bin** directory, containing the path to the font file you want to use. It looks like
+Finally, you can optionally create a 'freerct.cfg' INI format file next to the 'freerct' program in the **bin** directory, containing the settings to use. All entries are optional. It looks like
```

Comment:
Perhaps mention default values as well?

Always useful if you can read what to expect when lazy :)